src/main/scala/io/qbeast/spark/delta/OTreeIndex.scala

```diff
@@ -80,17 +100,17 @@ case class OTreeIndex(index: TahoeLogFileIndex) extends FileIndex {
   override def partitionSchema: StructType = index.partitionSchema
 }
 
-/**
- * Object OTreeIndex to create a new OTreeIndex
- * @param sparkSession the spark session
- * @param path the path to the delta log
- * @return the OTreeIndex
- */
 object OTreeIndex {
 
   def apply(spark: SparkSession, path: Path): OTreeIndex = {
     val deltaLog = DeltaLog.forTable(spark, path)
-    val tahoe = TahoeLogFileIndex(spark, deltaLog, path, deltaLog.snapshot, Seq.empty, false)
+    val tahoe = TahoeLogFileIndex(
```

Comment:
Is necessary to specify `isTimeTravelQuery`? Since we are not using that for the other parameters, we can leave it as in the main version.